doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -768,24 +768,18 @@ Modem libraries
 
 * :ref:`lte_lc_readme` library:
 
-  * Removed:
-
-    * The :c:func:`lte_lc_init` function.
-      All instances of this function can be removed without any additional actions.
-    * The :c:func:`lte_lc_deinit` function.
-      Use the :c:func:`lte_lc_power_off` function instead.
-    * The :c:func:`lte_lc_init_and_connect` function.
-      Use the :c:func:`lte_lc_connect` function instead.
-    * The :c:func:`lte_lc_init_and_connect_async` function.
-      Use the :c:func:`lte_lc_connect_async` function instead.
-    * The ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
-      Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
-      In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
-
-  * Deprecated the :c:macro:`LTE_LC_ON_CFUN` macro.
-    Use the :c:macro:`NRF_MODEM_LIB_ON_CFUN` macro instead.
+  * Added:
 
-  * Added a new :c:enum:`LTE_LC_EVT_RAI_UPDATE` event that is enabled with the :kconfig:option:`CONFIG_LTE_RAI_REQ` Kconfig option.
+    * The new :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option to enable the Connection Parameters Evaluation module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_EDRX` Kconfig option to enable the eDRX module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_NEIGHBOR_CELL_MEAS` Kconfig option to enable the Neighboring Cell Measurements module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_PERIODIC_SEARCH` Kconfig option to enable the Periodic Search Configuration module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_PSM` Kconfig option to enable the PSM module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_RAI` Kconfig option to enable the RAI module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_MODEM_SLEEP` Kconfig option to enable the Modem Sleep Notifications module.
```

Comment:
I am a little confused -- `Kconfig option to enable the Modem Sleep Notifications module.` but then what is `CONFIG_LTE_LC_MODEM_SLEEP_NOTIFICATIONS`?

Reply:
Explained in https://github.com/nrfconnect/sdk-nrf/pull/17266#discussion_r1803738642. I guess this comment could be resolved as the discussion is in the other thread.